joomla/joomla-cms
# [4.0] Use correct jQuery minicolours NPM package and update

### Summary of Changes

The current NPM package being pulled for jQuery Minicolours is incorrect. This PR addresses that and also updates to the latest version (v2.2.5)

### Testing Instructions

- Apply PR:
- Add a colour field manually or go to [this line](https://github.com/joomla/joomla-cms/blob/4.0-dev/templates/aurora/templateDetails.xml#L58) and change to `type="color"`
- Go to Templates >> Styles >> Aurora
- Ensure the colour picker works correctly
